profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -826,3 +914,99 @@ function ecas_get_user_department($user_info) {
 
   return array();
 }
+
+/**
+ * Gets the error message value for the specified variable.
+ *
+ * @param string $variable_name
+ *   The name of the variable related to the error message.
+ *
+ * @return array|bool
+ *   The error message value; or FALSE if not found.
+ */
+function _ecas_get_error_message_value($variable_name) {
```

Comment:
I don't agree with this approach but since it's not a technical opinion (it's an opinion about the overall strategy that affects the all solution), I will give my opinion in Jira ticket.

Reply:
I will comment in the ticket for this part.